driver/examples/configs/baroclinic_c12.yaml

```diff
@@ -6,6 +6,10 @@ stencil_config:
   device_sync: true
 initialization_type: baroclinic
 initialization_config: {}
+performance_config:
+  timer: true
+  json_dump: true
+  experiment_name: c12_baroclinic
```

Comment:
why is the experiment name specific to the performance config?

Reply:
This was a suggestion by Jeremy because experiment name is only used inside performance config. We do not use `input.yml`, the only time that comes into play is if we want to initialize with serialbox (which is expected to only be used internally).